.gitlab-ci.yml

```diff
@@ -616,9 +616,14 @@ test_backend_integration:
       done
   script:
     - cd integration/backend-tests/
+    # for older releases, ignore test suite selection and just run open tests
     - if [ -f ../docker-compose.enterprise.yml ]; then
-      PYTEST_ARGS="-k 'not Enterprise and not Multitenant'" ./run &&
-      PYTEST_ARGS="-k Enterprise" ./run -f=../docker-compose.enterprise.yml -f=../docker-compose.storage.minio.yml;
+      case $INTEGRATION_TEST_SUITE in
```

Comment:
what's missing here is the 
`- export INTEGRATION_TEST_SUITE=$(integration/extra/release_tool.py --select-test-suite || echo "all")`

statement (see the integration test def - you didn't forget it there).

without it, all tests will be ran always.

I started a few test pipelines - they are taking forever because I failed to unset a couple opts - but I can already see backend integration is not selected.

will keep you posted on the other ones.

Reply:
Thanks. Good catch.

Btw, I don't think they need to be exported (nor here nor in the yoctobuild script). But it doesn't hurt and better to keep both recipes aligned

Reply:
this one looks good. but I re-ran my tests after you amended and... still both sets of tests run in case of `backend-integration`.

could there be a timing issue here, does gitlab have some caches? I'm looking at your PR and can't see it not working.

Reply:
disregard^^, I screwed up the PR selection this time